lms/static/js/student_account/views/AccessView.js

```diff
@@ -1,19 +1,11 @@
 var edx = edx || {};
 
-(function($, _, _s, Backbone, gettext) {
+(function($, _, _s, Backbone, History) {
     'use strict';
 
     edx.student = edx.student || {};
     edx.student.account = edx.student.account || {};
 
-    // Bind to StateChange Event
-    History.Adapter.bind( window, 'statechange', function() {
```

Comment:
I did a little more digging into the event handler, and I agree that it's not necessary. [`getState()`](https://github.com/browserstate/history.js/#states) doesn't appear to have any side effects.